app/src/main/java/com/beemdevelopment/aegis/otp/GoogleAuthInfo.java

```diff
@@ -89,12 +89,13 @@ public static GoogleAuthInfo parseUri(Uri uri) throws GoogleAuthInfoException {
                     info = hotpInfo;
                     break;
                 case YandexInfo.HOST_ID:
+                case MotpInfo.ID:
```

Comment:
I'm a little confused to see that we're encoding mOTP secrets as hexadecimal everywhere, except for the QR codes. Is this the convention for mOTP?

Reply:
You're right the secrets are hex, will update this. my use case of pfSense wontwon't generate QR codes for MOTP whereas it does for google authenticator, so the only use case I tested for QR codes was Aegis to Aegis

Reply:
Good find. It also looks like the PIN should not be included in the URI, according to that spec.